Livecheckables/qd.rb

```diff
@@ -1,4 +1,4 @@
 class Qd
-  livecheck :url   => "http://crd-legacy.lbl.gov/~dhbailey/mpdist/",
+  livecheck :url   => "https://crd-legacy.lbl.gov/~dhbailey/mpdist/",
```

Comment:
```suggestion
  livecheck :url   => "https://www.davidhbailey.com/dhbsoftware/",
```

This redirects to https://www.davidhbailey.com/dhbsoftware/, so we should update the URL to avoid the redirection.